cloud/edgecontroller/pkg/controller/controller/downstream.go

```diff
@@ -41,6 +44,9 @@ func (dc *DownstreamController) syncPod(stop chan struct{}) {
 				log.LOGGER.Warnf("object type: %T unsupported", pod)
 				continue
 			}
+			if role := dc.lc.GetNodeRole(pod.Spec.NodeName); role != "edge" {
```

Comment:
What are the valid roles?

Reply:
I don't think Kubernetes has a restriction on the value of role since it is a label.  But we only process the update of the nodes whose role is edge. Hope the solution looks ok ?